projects/plugins/crm/modules/automations/jpcrm-automations-init.php

```diff
@@ -0,0 +1,118 @@
+<?php
+/**
+ * Jetpack CRM
+ * https://jetpackcrm.com
+ *
+ * Automation Module initialization
+ *
+ * @package automattic/jetpack-crm
+ */
+
+namespace Automattic\Jetpack_CRM\Automations;
+
+if ( ! apply_filters( 'jetpack_crm_feature_flag_automations', false ) ) {
+	return;
+}
+
+if ( ! defined( 'ZEROBSCRM_PATH' ) ) {
+	exit;
+}
+
+use Automattic\Jetpack\Assets;
+
+/**
+ * The main initializing function.
+ *
+ * @return void
+ */
+function init() {
+	add_action( 'admin_menu', 'Automattic\Jetpack_CRM\Automations\register_page' );
+	add_action( 'load-admin_page_jpcrm-automations', 'Automattic\Jetpack_CRM\Automations\admin_init' );
+}
+
+/**
+ * Register the admin page for the Automations UI
+ *
+ * @return void
+ */
+function register_page() {
+	global $zbs;
+
+	add_submenu_page(
+		$zbs->slugs['automations'],
+		__( 'Welcome to Jetpack CRM Automations', 'zero-bs-crm' ), // TODO: change this
```

Comment:
I put the TODO here as I'm not sure what this copy should be, so this is just a reasonable placeholder.

Reply:
The existing routes just say e.g. `Quotes`.
Maybe it's about time we update it to prefix it with e.g.: `Jetpack CRM: `, so in this case it would just be `Jetpack CRM: Automations`.

The main objective here is to make it easy for folks to navigate their browser tabs and doesn't have to apply to any SEO related rules

Reply:
Once I got the menu to work I found that this title is what is used in the menu:

<img width="163" alt="image" src="https://github.com/Automattic/jetpack/assets/42627630/2125a3d4-b72a-4da5-b208-7ffaa71cf757">

So I figured it is best as Automations.